docker/docker-compose.yml

```diff
@@ -46,6 +46,10 @@ services:
       LOGFLARE_API_KEY: ${LOGFLARE_API_KEY}
       LOGFLARE_URL: http://analytics:4000
       NEXT_PUBLIC_ENABLE_LOGS: true
+      # Comment to use Big Query backend for analytics
+      NEXT_ANALYTICS_BACKEND_PROVIDER: postgres
```

Comment:
AFAIK this env var is not dynamic for the cli, so the error will not show for cli.

Reply:
cc @sweatybridge

Reply:
I will need to change the cli PR to add this env var during build time